lib/hammer_cli_katello/repository.rb

```diff
@@ -75,6 +76,7 @@ class InfoCommand < HammerCLIKatello::InfoCommand
       def extend_data(data)
```

Comment:
you may want to do a bit of refactoring on this method to reduce cyclomatic complexity for rubocop